azblob/zt_sas_blob_snapshot_test.go

```diff
@@ -0,0 +1,104 @@
+package azblob_test
+
+import (
+	"bytes"
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	chk "gopkg.in/check.v1"
+	"strings"
+	"time"
+)
+
+func (s *aztestsSuite) TestSnapshotSASUsage(c *chk.C) {
+	//Generate URLs ----------------------------------------------------------------------------------------------------
+	bsu := getBSU()
```

Comment:
Great test!

Minor: I'd suggest to make the test more concise by using Shared Key for the set up and taking out the part where Account SAS is used, and make that a test for the account SAS instead.